src/index.js

```diff
@@ -1 +1,75 @@
+/* eslint-disable no-console */
 'use strict';
+
+const http = require('http');
+const fs = require('fs');
+const multiparty = require('multiparty');
+const { getCompressionType } = require('./getCompressionType');
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((req, res) => {
+  if (req.method === 'POST') {
+    const form = new multiparty.Form();
+
+    form.parse(req, (err, fields, files) => {
+      if (err) {
+        res.statusCode = 500;
```

Comment:
The server error message is too generic. It's recommended to provide more detailed error messages or log the error for debugging purposes.